Bottom: 78163708680ad1a0e2e3313a753bd6f37aa59ec3
Top:    3f4bb892a5a463c5e1aaa33760b1399433180122
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-05 17:24:28 +0100

Refresh of move-arc-from-included-angle

---

diff --git a/src/misc.c b/src/misc.c
index 703e8dc..59d3336 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -366,7 +366,13 @@ calc_arc_from_points_and_included_angle (PointType *p1, PointType *p2, Angle inc
   *cx = hx + unit_hcx * c_to_h_dist;
   *cy = hy + unit_hcy * c_to_h_dist;
 
-#warning radius, start_angle and delta_angle are not computed yet!
+  if (radius != NULL)
+    {
+      *radius = sqrt ((p1->X - *cx) * (p1->X - *cx) +
+                      (p1->Y - *cy) * (p1->Y - *cy));
+    }
+
+#warning start_angle and delta_angle are not computed yet!
 }
 
 /* ---------------------------------------------------------------------------
